src/components/Avatar/Avatar.styles.ts

```diff
@@ -10,20 +10,22 @@ export interface AvatarStyleProps {
   size: AvatarSize;
 }
 
-export const getPersonaCoinStyles = memoizeFunction((props: AvatarStyleProps): (() => Partial<IPersonaCoinStyles>) => {
-  const { borderType, size } = props;
-  const borderRadius = borders[borderType];
-  const initials = size === AvatarSize.XLARGE ? { lineHeight: '70px' } : {};
-  return () => ({
-    initials,
-    image: {
-      borderRadius,
-    },
-    imageArea: {
-      borderRadius,
-    },
-  });
-});
+export const getPersonaCoinStyles = memoizeFunction(
```

Comment:
seems that the rules in .prettierrc aren't being respected by the git hook anymore

Reply:
I've bumped prettier back down